mainwindow.cpp

```diff
@@ -465,7 +466,7 @@ bool MainWindow::eventFilter(QObject *watched, QEvent *event)
     QWidget* widged = (QWidget*) watched;
     switch (event->type())
     {
-    case QEvent::StyleAnimationUpdate:
+    case 1://QEvent::StyleAnimationUpdate:
```

Comment:
Just wrap around an #ifdef !OSX. I don't expect anyone to attempt building with 4.6.2 on OS X.